src/Collection.php

```diff
@@ -282,8 +282,8 @@ public function count($filter = [], array $options = [])
      */
     public function createIndex($key, array $options = [])
     {
-        $indexOptions = array_diff_key($options, ['writeConcern' => 1]);
-        $commandOptions = array_intersect_key($options, ['writeConcern' => 1]);
+        $indexOptions = array_diff_key($options, ['maxTimeMS' => 1, 'writeConcern' => 1]);
+        $commandOptions = array_intersect_key($options, ['maxTimeMS' => 1, 'writeConcern' => 1]);
```

Comment:
In #432 you've also added a test case to test for invalid values for the new option. Can you add one for this new option to ``provideInvalidConstructorOptions`` in ``CreateIndexesTest.php`` too?

Reply:
Done, thanks!